tasks/util/update_version.js

```diff
@@ -11,7 +11,7 @@ module.exports = function updateVersion(pathToFile) {
             if(isVersionNode(node)) node.update('\'' + pkg.version + '\'');
         });
 
-        fs.writeFile(pathToFile, out, function(err) {
+        fs.writeFile(pathToFile, out.toString(), function(err) {
```

Comment:
Let's test for the type of `out` to be `object` before casting to string.

Reply:
Hey! Sorry for a long answer. I don't think that it's necessary. It doesn't matter what `out` is. It inherit `toString()` method from `Object`. Even if it's string it has a `toString` method.